docs/get-started/getting-started-workload-qualification.md

```diff
@@ -0,0 +1,135 @@
+---
+layout: page
+title: Workload Qualification
+nav_order: 7
+parent: Getting-Started
+---
+# Getting Started on Spark workload qualification
+
+The RAPIDS Accelerator for Apache Spark does not support all the features which are supported in Apache Spark.
+If you plan to convert existing Spark workload from CPU to GPU, it is highly recommended that you do gap analysis 
+to figure out if there are any unsupported features such as functions, expressions, data types, data formats.
+After that we will know which workload is more suitable to GPU, which is called "workload qualification" here.
+
+If there are certain operators which can not run on GPU due to the current limitations, they will fallback to CPU mode.
+As a result, it may incur some performance overhead because of host memory <=> GPU memory transfer. 
+
+This article is to help you get familiar with the best practice and different tools we provide on how to do gap analysis 
+and workload qualification.
+
+## 1. Qualification and Profiling tool
+
+### Requirement
+
+- Access to Spark event logs from Spark 2.x or 3.x
+- Spark 3.0.1+ jars
+- `rapids-4-spark-tools` jar
+
+### How to use
+
+If you have old Spark event logs from Spark 2.x or 3.x, you can use [Qualification tool](../spark-qualification-tool.md) 
+and [Profiling tool](../spark-profiling-tool.md) to analyze them.
+Qualification tool can output the score, rank and some of the potential not-supported features for each Spark application.
+Its output can help you focus on the top N Spark applications which are SQL heavy applications. 
+
+Profiling tool can output SQL plan metrics and also print out actual query plans to provide more insights.
+For example, below Profiling tool output for a specific Spark application shows that it has a query with a large 
+`HashAggregate` and `SortMergeJoin`. Those are indicators for a good candidate for RAPIDS accelerator.
+
+```
++--------+-----+------+----------------------------------------------------+-------------+------------------------------------+-------------+----------+
+|appIndex|sqlID|nodeID|nodeName                                            |accumulatorId|name                                |max_value    |metricType|
++--------+-----+------+----------------------------------------------------+-------------+------------------------------------+-------------+----------+
+|1       |88   |8     |SortMergeJoin                                       |11111        |number of output rows               |500000000    |sum       |
+|1       |88   |9     |HashAggregate                                       |22222        |number of output rows               |600000000    |sum       |
+```
+
+Since the 2 tools are basically analyzing Spark event logs, it can not provide very accurate gap analysis information.
+But it is very convenient because you do not need a CPU or GPU Spark cluster to run them.
```

Comment:
```suggestion
Since the two tools are only analyzing Spark event logs, it can not provide very accurate gap analysis information.
However it is very convenient because you do not need a CPU or GPU Spark cluster to run them.
```